package.json

```diff
@@ -55,5 +55,11 @@
     "mocha": "^2.4.5",
     "rx": "^4.0.7",
     "uglifyify": "^3.0.1"
+  },
+  "peerDependencies": {
```

Comment:
Sure thing, I wasn't even aware they were a thing until I rummaged around the http driver to see how it managed to not have this as a dependency.